releasenotes/notes/logs_docker_environment_variable-daf65e8291a6d309.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Added an environment variable `DD_LOGS_CONFIG_COLLECT_ALL` to enable logs tailing on all containers.
```

Comment:
you haven't updated the release note with the updated config option name ;)

Reply:
Good catch, thanks